### PR TITLE
Guidelines for multi-line function call formatting

### PR DESCRIPTION
**Pull request by jeffblack360**
Add guidelines for multi-line function call formatting for future contributors.

### CODE REVIEW

File: en/contributing/cakephp-coding-conventions.rst [hunk 1]
```diff
@@ -45,6 +45,27 @@ Or::
...
+                        array_keys($this->_listeners), 0)));
+
+Use this instead::
+
+    $matches = array_intersect_key(
+        $this->_listeners,
+        array_flip(
+            preg_grep($matchPattern, array_keys($this->_listeners), 0)
+            )
+        );
```
Comment (dereuromark) [inline en/contributing/cakephp-coding-conventions.rst]: The closing brackets should both be a level up (symmetrical to the opening ideally).

Comment (jeffblack360) [inline en/contributing/cakephp-coding-conventions.rst]: @dereuromark Thanks!  I got carried away with my shifts in Vim.

The following code is what I passed Travis in my earlier CakePHP PR.

    $matches = array_intersect_key(
        $this->_listeners,
        array_flip(
            preg_grep($matchPattern, array_keys($this->_listeners), 0)
        )
    );

**Comment by jeffblack360** [comment]
I don't see this documentation change "live" yet. Has it not been released?

**Comment by saeideng** [comment]
you send this PR in 2.x version
http://book.cakephp.org/2.0/en/contributing/cakephp-coding-conventions.html#indentation
master branch is for 2.x
branch 3.0 for 3.x

**Comment by jeffblack360** [comment]
Oh dear. That was a goof.

**Pull request merged**